src/quest_manager/templates/quest_manager/buttons.html

```diff
@@ -8,7 +8,15 @@
       </a>
     {% endif %}
 
-    {% if  request.user.is_staff or request.user == q.editor %}
+    {% if q.archived and request.user.is_staff %}
+      <form action="{% url 'quests:unarchive' q.id %}" method="post" style="display:inline">
```

Comment:
We shouldn't be using inline styles, just use the appropriate bootstrap styling or the style classes already used elsewhere in the project for equivalent stuff.

If it's because the form is causing the buttons to look off, look at the other places in the project where we mix POST form buttons and GET (url link) buttons.